esp/services/ws_fs/ws_fsService.cpp

```diff
@@ -3504,6 +3542,9 @@ bool CFileSprayEx::onDeleteDropZoneFiles(IEspContext &context, IEspDeleteDropZon
         if (!checkDropZoneIPAndPath(version, dzName, netAddress, path.str()))
             throw MakeStringException(ECLWATCH_DROP_ZONE_NOT_FOUND, "Dropzone is not found in the environment settings.");
 
+        if (getDropZoneScopePermissions(context, dzName, netAddress, path) < SecAccess_Full)
```

Comment:
minor: this should prob. go after the  if(f->isDirectory()!=fileBool::foundYes) check (on line 3555)

Reply:
I thought about that. Should we expose the 'cannot resolve network IP' error or the 'not a directory' error if the user does not have the scope access?

Reply:
thinking about it, I think prob. more correct to check perms. 1st, before validating if a dir, or if host resolvable.